### PR TITLE
Adding PostgreSQL DB slow query log 

### PR DESCRIPTION
**Pull request by cwarnermm**
Troubleshooting for high-scale deployments

### CODE REVIEW

File: source/deploy/high-scale-troubleshoot.rst [hunk 1]
```diff
@@ -12,4 +12,18 @@ My system keeps hanging when I search for a message in Mattermost
...
\ No newline at end of file
+Go to the **System Console > Reporting > Site Statistics** and review the **Total Posts** figure reported. If this figure is above 3,000,000 posts, we recommend deploying Elasticsearch alongside your Mattermost server for improved search performance. Follow our guides to `deploy an Elasticsearch server </scale/elasticsearch.html>`__.
+
+How to troubleshoot server-side performance issues without Prometheus or Grafana?
+-----------------------------------------------------------------------------------
+
+Enable slow query logging in PostgreSQL and leave it enabled to gather data over time. PostgreSQL's slow query log helps you identify queries that take longer than a specified amount of time. The slow query log isn't enabled by default, and must be enabled manually.
+
+To enable slow query logging globally, change the following line in postgresql.conf and reload the page: ``log_min_duration_statement = 5000``. When you this value to ``5000``, PostgreSQL considers queries that take longer than 5 seconds to be slow queries and logs them in the log file.
```
Comment (coltoneshaw) [inline source/deploy/high-scale-troubleshoot.rst:22]: I'd set our suggestion to 1s for this, just to capture more. Anything longer than 1s really should be looked at.

File: source/deploy/high-scale-troubleshoot.rst [hunk 2]
```diff
@@ -12,4 +12,18 @@ My system keeps hanging when I search for a message in Mattermost
...
+
+To enable slow query logging for a specific database, use ``ALTER DATABASE`` to change the configuration parameter for a single database. For example:
+
+.. code-block:: none
+
+  postgres=# ALTER DATABASE test SET log_min_duration_statement = 5000;
```
Comment (coltoneshaw) [inline source/deploy/high-scale-troubleshoot.rst:28]: let's change `test` to `mattermost` and make a call out that the database name should be what you call it. 99% will be `mattermost`

File: source/deploy/high-scale-troubleshoot.rst (hunk 1 above)
Comment (coltoneshaw) [inline source/deploy/high-scale-troubleshoot.rst:22]: also, doing it this way requires a restart or a config reload. We have it reading as `reload the page` but it should be `restart postgres` which causes a short outage. 

Additionally, we could just have them run `SELECT pg_reload_conf();` to reload the config without a restart.

**Review by coltoneshaw — CHANGES_REQUESTED** [review]

File: source/deploy/high-scale-troubleshoot.rst [hunk 3]
```diff
@@ -12,4 +12,18 @@ My system keeps hanging when I search for a message in Mattermost
...
+  ALTER DATABASE
```
Comment (coltoneshaw) [inline source/deploy/high-scale-troubleshoot.rst:29]: This actually should be removed, it's just a hanging statement if used in this way.

**Comment by cwarnermm** [comment]
Thanks @coltoneshaw! This docs PR is now ready for your final review.

**Review by coltoneshaw — APPROVED** [review]

**Pull request merged**